sky/tools/create_macos_framework.py

```diff
@@ -152,6 +157,29 @@ def process_framework(dst, args, fat_framework, fat_framework_binary):
     ],
                           cwd=dst)
 
+  macos_filepath_with_entitlements = ''
+  macos_filepath_without_entitlements = 'FlutterMacOS.framework/Versions/A/FlutterMacOS'
+
+  embed_codesign_configuration(
+      os.path.join(dst, 'entitlements.txt'), macos_filepath_with_entitlements
+  )
+
+  embed_codesign_configuration(
+      os.path.join(dst, 'without_entitlements.txt'),
+      macos_filepath_without_entitlements
+  )
+
+  # Zip FlutterMacOS.framework.
+  subprocess.check_call([
+      'zip',
+      '-r',
```

Comment:
include `y` option?

Reply:
+1 this needs to preserve symlinks.

Reply:
Yeah we talked about this yesterday night. Haven't updated here in github yet. But right the -y flag is needed to preserve the symlink of a child file. Previously I thought only the framework zip needs the y flag and didn't know the outer most layer of zip also need the -y flag.